src/ExposureHistory/History.tsx

```diff
@@ -64,7 +72,11 @@ const History: FunctionComponent<HistoryProps> = ({ lastDetectionDate }) => {
           </View>
         </View>
         <View style={styles.listContainer}>
```

Comment:
I think that this container and it's style might be better of inside the `NoExposures` and the `ExposuresList`, I feel like those will have very different style needs. Is probably fine but we should keep an eye for overriding this style inside those components, might be a sign that we don't need this style.